css/css.go

```diff
@@ -1201,6 +1201,35 @@ func (c *cssMinifier) minifyProperty(prop Hash, values []Token) []Token {
 			values[0].Data = oneBytes
 			values[0].Ident = 0
 		}
+	case UnicodeRange:
+		for i, value := range values {
+			// Copy data so can mutate without having to undo.
+			data := make([]byte, len(value.Data))
+			copy(data, value.Data)
+
+			// If we have a range of exactly two parts.
+			if parts := bytes.Split(data, []byte{'-'}); len(parts) == 2 {
```

Comment:
I would remove `U+` first, for speed just check if the first byte is `U` and the second a `+`, then take a slice starting at `2` to skip them both. Next we have two cases: one single value or a range. Start looking byte for byte until we meet the end or `-`. Keep track of leading `0`s so we trim those too. If this is a range, continue like you do.